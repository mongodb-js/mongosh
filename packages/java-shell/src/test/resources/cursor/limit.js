// before
db.coll.deleteMany({});
db.coll.insertOne({a: 1});
db.coll.insertOne({a: 2});
db.coll.insertOne({a: 3});
// command
db.coll.find().limit(2);
// clear
db.coll.drop();