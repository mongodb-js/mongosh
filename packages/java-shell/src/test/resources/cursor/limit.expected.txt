[ { "_id": <ObjectID>, "a": 1 }, { "_id": <ObjectID>, "a": 2 } ]